core/pom.xml

```diff
@@ -30,6 +30,10 @@
          <artifactId>jgroups</artifactId>
       </dependency>
 
+      <dependency>
```

Comment:
Unfortunately this won't solve the problem on client side... one can add `infinispan-core` as a library, use `default-jgroups-kubernetes.xml` (it's already in core after all!) and his application will crash (because there is no `KUBE_PING` protocol on the classpath).